### DISCUSSION TITLE
TailwindCSS

### DISCUSSION THREAD

**Discussion by Yash-joyride** [react-boilerplate/react-boilerplate#3066] [Q&A]
I've installed TailwindCSS (Ref - https://tailwindcss.com/docs/guides/create-react-app) , I've followed all the steps...
But it's not working please help me i want to know where to add below things 

@tailwind base;
@tailwind components;
@tailwind utilities;

OR
Guide me to setup the tailwindCSS

**Comment by shiv123-max** [comment]
Have you got the answer to this? If yes, please email me. I am also facing the same issue.

**Reply by Yash-joyride** [reply]
I also don't get any answer, Yes but I'm using TailwindCSS's CDN for temporary solution...
@shiv123-max

**Comment by CeoFred** [comment]
![image](https://user-images.githubusercontent.com/32965534/191312653-3b549a73-9bb8-42d8-bf03-9bb7914f08a1.jpg)
So don’t forget to use that content key in the exported module .. it should be a path to where your react files are

**Reply by shiv123-max** [reply]
I am doing the same thing. but it's not working
![Screenshot (52)](https://user-images.githubusercontent.com/56493775/191314278-ab3be354-c4b2-4a93-9fe2-19dd8b2d463d.png)
Above is my folder structure 'app' directory has all the files where I wan to use Tailwind

**Comment by CeoFred** [comment]
You should install tailwind inside your app folder and have the
configuration in there too

**Reply by shiv123-max** [reply]
Still ain't working

**Reply by RymOueslati** [reply]
Can you send a demo code to a repo git please?
